src/components/filters/FilterTimeFrameSelect.jsx

```diff
@@ -91,14 +110,22 @@ export default class FilterTimeFrameSelect extends React.Component {
                 after = todayStr;
                 before = today.addDays(30).format('{yyyy}-{MM}-{dd}');
                 break;
+            case 'inlast':
+                days = this.state.days || '30';
+                after = `-${days}d`;
```

Comment:
I'm wondering if we should add a `before` here as well. The label implies that matches will be found from the last 30 days, starting 30 days ago until today, but the actual search is for anything after 30 days ago (i.e. also in the future). This is only relevant for campaign participation filter. But maybe it would make sense?

Based on your understanding from having worked with this feature, would that be difficult? What would happen to any old queries that lack a `before` when they're opened in the interface if we add `before`. It's only added after save, right?

Reply:
You mean like before='now'? No I don't think it would be difficult, but yes, any queries already created would have the old behavior.

Reply:
Exactly! Do you want to make that change before we merge?